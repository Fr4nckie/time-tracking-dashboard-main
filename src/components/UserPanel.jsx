import '../styles/user-panel.scss'
import TimeFrameSelector from './TimeFrameSelector.jsx'
import UserProfile from './UserProfile.jsx'
function UserPanel() {
    return (
        <div className="user-panel">
            <UserProfile />
            <TimeFrameSelector />
        </div>
    )
}


export default UserPanel
