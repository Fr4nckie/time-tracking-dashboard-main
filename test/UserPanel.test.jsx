import { render } from '@testing-library/react'
import { describe, expect, it } from 'vitest'
import UserPanel from '../src/components/UserPanel.jsx'

describe('<UserPanel />', () => {
    it('Should render correctly', () => {
        const { container } = render(<UserPanel />)
        expect(container.firstChild).toMatchSnapshot()
    })
})
