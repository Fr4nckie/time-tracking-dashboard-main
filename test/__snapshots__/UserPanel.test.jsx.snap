// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<UserPanel /> > Should render correctly 1`] = `
<div
  class="user-panel"
>
  <div
    class="profile"
  >
    <img
      alt="profile pic"
      class="profile__avatar"
      src="/src/assets/image-jeremy.png"
    />
    <div
      class="profile__name"
    >
      <p>
        Report for
      </p>
      <h1>
        Jeremy Robson
      </h1>
    </div>
  </div>
  <ul
    class="selector"
  >
    <li
      class="null"
      role="button"
    >
      daily
    </li>
    <li
      class="active"
      role="button"
    >
      weekly
    </li>
    <li
      class="null"
      role="button"
    >
      monthly
    </li>
  </ul>
</div>
`;
